editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5454,15 +5454,15 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	group_button->set_flat(true);
 	hb->add_child(group_button);
 	group_button->connect("pressed", callable_mp(this, &CanvasItemEditor::_popup_callback), varray(GROUP_SELECTED));
-	group_button->set_tooltip(TTR("Makes sure the object's children are not selectable."));
+	group_button->set_tooltip(TTR("Make selected node's children not selectable."));
```

Comment:
```suggestion
	group_button->set_tooltip(TTR("Make the selected node's children unselectable."));
```

Reply:
I chose "not selectable" instead of "unselectable" to be consistent with this:
![image](https://user-images.githubusercontent.com/12120644/126198909-cd93f9c4-5f8a-455b-a9bb-9942d5e1b3e5.png)
I think it's easier to read at a glance as well.

And I didn't include a "the' to be consistent with the lock button right next to it, which doesn't the "the":
![image](https://user-images.githubusercontent.com/12120644/126212054-f2140ef5-8ea5-45d3-bfad-56f5bdeb902c.png)
Both of them should be changed if really desired

Reply:
I think adding "the" makes it sound more natural. I'm fine with "not selectable", I do think "Children are unselectable" sounds weirder than "Children are not selectable."